_posts/2018-08-19-openfaas-on-fargate.md

```diff
@@ -0,0 +1,281 @@
+```
+
+1\. Lets create and deploy a function to our new stack using the CLI:
+
+```shell
+$ faas new hello-world --lang go -p ewilde
+
+2018/12/22 10:29:03 No templates found in current directory.
+2018/12/22 10:29:03 Attempting to expand templates from https://github.com/openfaas/templates.git
+2018/12/22 10:29:05 Fetched 14 template(s) : [csharp dockerfile go go-armhf java8 node node-arm64 node-armhf php7 python python-armhf python3 python3-armhf ruby] from https://github.com/openfaas/templates.git
+Folder: hello-world created.
+  ___                   _____           ____
+ / _ \ _ __   ___ _ __ |  ___|_ _  __ _/ ___|
+| | | | '_ \ / _ \ '_ \| |_ / _` |/ _` \___ \
+| |_| | |_) |  __/ | | |  _| (_| | (_| |___) |
+ \___/| .__/ \___|_| |_|_|  \__,_|\__,_|____/
+      |_|
+
+
+Function created in folder: hello-world
+Stack file written: hello-world.yml
+```
+In the above command `-p` is my docker hub username where the image will get pushed to during the build process
+
+2\. Open `./hello-world/handler.go` to see the function entry point
+
+```golang
+package function
+
+import (
+	"fmt"
+)
+
+// Handle a serverless request
+func Handle(req []byte) string {
+	return fmt.Sprintf("Hello, Go. You said: %s", string(req))
+}
+```
+
+3\. Now lets build and deploy the function to our new environment
+```shell
+$ faas up -f hello-world.yml
+
+... <- lots of lovely build output from docker
+
+Successfully built f9833391f176
+Successfully tagged ewilde/hello-world:latest
+Image: ewilde/hello-world:latest built.
+[0] < Building hello-world done.
+[0] worker done.
+
+[0] > Pushing hello-world [ewilde/hello-world:latest].
+The push refers to repository [docker.io/ewilde/hello-world]
+d1307abdafe5: Mounted from ewilde/hello-paris
+9808cfceb9f2: Mounted from ewilde/hello-paris
+a0db5d04a9ae: Mounted from ewilde/hello-paris
+39c32d89aeac: Mounted from ewilde/hello-paris
+c198ae36274e: Mounted from ewilde/hello-paris
+b01940cbde37: Mounted from ewilde/hello-paris
+df64d3292fd6: Mounted from ewilde/env
+latest: digest: sha256:e0dbd58d58880e0771c3ab36cdcbd7a371fa80d91894f9e845f5f54132124586 size: 1785
+[0] < Pushing hello-world [ewilde/hello-world:latest] done.
+[0] worker done.
+
+Deploying: hello-world.
+
+Deployed. 202 Accepted.
+URL: https://openfaas.{your-domain}.com/function/hello-world
+```
+
+### Removing the installation
+* Remove any functions that you have deployed using `faas-cli remove`
+* Run `make destroy`. This can take between 5-10 minutes
+
+```shell
+$ make destroy
+
+... <--- lots of destroying going on here
+
+aws_subnet.external[2]: Destruction complete after 1m48s
+aws_security_group.alb: Still destroying... (ID: sg-0a84a6bd951bacb6d, 1m50s elapsed)
+aws_security_group.alb: Destruction complete after 1m51s
+aws_vpc.default: Destroying... (ID: vpc-0220284dd7ba01f58)
+aws_vpc.default: Destruction complete after 0s
+
+Destroy complete! Resources: 133 destroyed.
+```
+
+## What's next for OpenFaaS on Fargate
```

Comment:
This is a very sudden close to the post.

Could you add a conclusion section with 

* A summary of what you did
* What's next for faas-fargate and link to the repo. Mention if you welcome contributions?
* See the wrapping-up section of the last few official blogs on how to construct a call to action pointing back to the project Slack, Twitter and etc